src/physics/utils/physics_data.F90

```diff
@@ -56,21 +56,20 @@ integer function find_input_name_idx(stdname, use_init_variables, constituent_in
       integer                       :: idx
       ! to test read_from_file status
       logical                       :: is_read
+      logical                       :: is_constituent
 
       !Initialize function:
       find_input_name_idx = no_exist_idx
       constituent_index = no_exist_idx
+      is_constituent = .false.
 
       !First check if quantity is a constituent:
       call const_get_index(trim(stdname), find_input_name_idx, abort=.false., warning=.false.)
       if (find_input_name_idx < 0) then
          find_input_name_idx = no_exist_idx
       else
          constituent_index = find_input_name_idx
```

Comment:
While we are here, could we simplify this code block to the following, given that `find_inpute_name_idx` is already initialized to `no_exist_idx`:

```suggestion
      if (find_input_name_idx >= 0) then
         constituent_index = find_input_name_idx
```

Reply:
ah good call! fixed!